rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -1056,11 +1088,21 @@ void getProposedFederation_whenProposedFederationIsCached_shouldReturnCachedFede
             federationStorageProvider.getProposedFederation(federationConstants, allActivations);
 
             // saving in the repo another fed to make sure the cached value is the one being returned
-            Federation anotherFederation = new StandardMultiSigFederationBuilder().build();
+            Federation anotherFederation = createAnotherFederation();
             bridgeStorageAccessor.saveToRepository(PROPOSED_FEDERATION_FORMAT_VERSION.getKey(), anotherFederation.getFormatVersion(), BridgeSerializationUtils::serializeInteger);
             bridgeStorageAccessor.saveToRepository(PROPOSED_FEDERATION.getKey(), anotherFederation, BridgeSerializationUtils::serializeFederation);
 
-            assertEquals(Optional.of(proposedFederation), federationStorageProvider.getProposedFederation(federationConstants, allActivations));
+            Optional<Federation> actualProposedFederation = federationStorageProvider.getProposedFederation(federationConstants, allActivations);
+            assertEquals(Optional.of(proposedFederation), actualProposedFederation);
+        }
+
+        private Federation createAnotherFederation() {
```

Comment:
Pretty sure we have something like this in FederationTestUtils. Let's use that to avoid ending up with methods like this all over the code base